src/matter/session/secure/SecureChannelMessenger.ts

```diff
@@ -7,6 +7,20 @@
 import { MessageExchange } from "../../common/MessageExchange";
 import { GeneralStatusCode, ProtocolStatusCode, MessageType, SECURE_CHANNEL_PROTOCOL_ID } from "./SecureChannelMessages";
 import { ByteArray, DataReader, DataWriter, Endian, TlvSchema } from "@project-chip/matter.js";
+import { MatterError } from "../../../error/MatterError";
+
+/** Error base Class for all errors related to the status response messages. */
+export class ChannelStatusResponseError extends MatterError {
```

Comment:
This is never catch anywhere...
Any reason to introduce this typed Error?

Reply:
Used in coming PRs